admin/src/hooks/use-preview-url.js

```diff
@@ -1,15 +1,16 @@
 import { useEffect, useState } from 'react';
-import qs from 'qs';
 
 import { usePluginConfig} from '../hooks';
-import { parseUrl, pluginId } from '../utils';
+import { parseUrl } from '../utils';
 
 const usePreviewUrl = ( uid, data, isDraft, isCreating ) => {
   const [ url, setUrl ] = useState( null );
   const [ canCopy, setCopy ] = useState( true );
   const { config, isLoading } = usePluginConfig();
   const { contentTypes } = config;
 
```

Comment:
@Dupflo Can you remove this empty line between the `config` and `data` destructuring?

Reply:
@mattmilburn done